README.md

```diff
@@ -1,22 +1,29 @@
-# stac-pydantic ![tests](https://github.com/arturo-ai/stac-pydantic/workflows/cicd/badge.svg)
-[Pydantic](https://pydantic-docs.helpmanual.io/) models for [STAC](https://github.com/radiantearth/stac-spec) Catalogs, Collections, Items, and the [STAC API](https://github.com/radiantearth/stac-api-spec) spec.  Initially developed by [arturo-ai](https://github.com/arturo-ai).
+# stac-pydantic
+
+![tests](https://github.com/stac-utils/stac-pydantic/actions/workflows/cicd.yml/badge.svg)
```

Comment:
this wasn't working because (1) it was on the same line as the H1 tag and (2) it was both pointing to the wrong organization and had the wrong path to the badge.svg